src/components/date-picker/date-picker.stories.ts

```diff
@@ -203,6 +203,17 @@ export const britishLang_TestOnly = (): string =>
     )}
   </div>`;
 
+export const arabicLang_TestOnly = (): string =>
+  html`<div style="width: 400px">
+    ${create(
+      "calcite-date-picker",
+      createAttributes({ exceptions: ["lang", "value"] }).concat([
+        { name: "lang", value: "ar" },
+        { name: "value", value: "2022-08-11" }
```

Comment:
Maybe the value can be added to the existing `arabLangNumberingSystem_TestOnly` story so we don't need to create a new one?